src/usr/local/www/diag_reboot.php

```diff
@@ -117,8 +120,12 @@ function startCountdown() {
 
 $form = new Form(false);
 
-$help = 'Click "Normal reboot" to reboot the system immediately, "Reboot with Filessystem Check" to reboot and run filesystem check';
-$modeslist = ['Reboot' => 'Normal reboot', 'FSCKReboot' => 'Reboot with Filesystem Check'];
+$help = 'Select "Normal reboot" to reboot the system immediately';
+$modeslist = ['Reboot' => 'Normal reboot'];
+if (get_single_sysctl('hw.machine') != 'arm') {
```

Comment:
php_uname('m') is a better options here since it's less expensive

Reply:
ok, done